data/fro/Stroński_1906_EliasDeBarjolsIII/metadata.txt

```diff
@@ -0,0 +1,63 @@
+publicationType : "book"
+publicationTitle : "Le troubadour Elias de Barjols edition critique"
+publicationDate : "1906"
+publicationPlace : "Toulouse"
+publicationSeries : "Bibliothèque méridionale"
+publicationNum: "10"
+publicationStemmaNum: ""
+publicationAuthors : "Stroński, Stanisław"
+publicationPage : "44"
+publicationLink : ""
+workTitle : "III"
+workViaf : ""
+workOrigDate : "1200-1204"
+workOrigPlace : "Sault"
+workAuthor : "Elias de Barjols"
+workAuthorViaf : "231763501"
+workGenre : "lyric poetry"
+workLangCode : "fro"

```

Comment:
That's actually `pro` (Provençal, Old (to 1500);occitan, Old (to 1500)) -> metadata and super-folder change.